build/README.md

```diff
@@ -323,6 +327,20 @@ a PR to that effect would be greatly appreciated.
 
         -   Note that `VAR_DOCKER_IMAGE_NAME` is used in two places.
 
+    1.  `VAR_DOCKER_IMAGE_PULL_SECRET`: Secret name of the credentials to access
```

Comment:
What should this value be if no credentials are needed to access the image registry (default situation)?  It looks like we need instructions regarding uncommenting the line or not and replacing the value or not.

Reply:
Ben, please have a look to the latest change to see if it meets your expectation

Reply:
The baseline instruction is "replace all `VAR_*` instances with appropriate values", so I think we should say something like "If the image specified in `VAR_DOCKER_IMAGE_NAME` does not require authentication, then do not populate this instance and do not uncomment the line containing it"